docs/products/compute/compute-instances/guides/send-email/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Send Email from a Compute Instance"
+description: "Learn about Linode's email policy, how to lift email restrictions, and best practices to implement when sending email."
+published: 2023-04-28
+authors: ["Linode"]
+---
+
+Linode Compute Instances can be used to send and receive email, as long as their are no restrictions in place (see [Email Policy and Restrictions](#email-policy-and-restrictions)). You may want to send and receive email on your Compute Instances if you wish to self-host your email server or email delivery tool. This guide covers our email policies and how to lift email restrictions from your account. It also includes best practices for both ensuring your email is compliant with regulations and to decrease the likelihood of having email marked as spam.
+
+{{< note >}}
+Managing an email server can be complicated and, if things aren't properly configured, can result in email deliverability issues. Many organizations choose to instead use dedicated external mail servers, such as those provided by [Fastmail](https://www.fastmail.com/), [Gmail for Google Workspace](https://workspace.google.com/products/gmail/), and [Outlook for Microsoft 365](https://www.microsoft.com/en-us/microsoft-365/business/compare-all-microsoft-365-business-products?&activetab=tab:primaryr2). There are also email delivery services for marketing and transactional email, including [Postmark](https://postmarkapp.com/), [SendGrid](https://sendgrid.com/), and [Mailchimp](https://mailchimp.com/). If you are using one of these services, you may not need to configure your Compute Instances to send email. Consult with those on your team that use email to determine if an external service better suits your needs.
+{{< /note >}}
+
+## Email Policy and Restrictions
+
+Email can be sent on the Linode Platform for any purpose (including marketing and transactional), provided that your use conforms to our [Acceptable Use Policy (AUP)](https://www.linode.com/legal-aup/) and any government policies that are applicable to you, including the [CAN-SPAM Act](https://www.ftc.gov/business-guidance/resources/can-spam-act-compliance-guide-business).
+
+In an effort to fight spam, Linode restricts outbound connections over ports 25, 465, and 587 on Compute Instances for *some* new accounts created after November 5th, 2019. These restrictions, if applied to your account, prevent your Compute Instance from sending email over the standard SMTP ports.
```

Comment:
How does a user know their ports are being blocked?

Reply:
Talked outside of GitHub. Leaving this as is for now.